internal/pkg/agent/cmd/enroll.go

```diff
@@ -351,11 +377,13 @@ func enroll(streams *cli.IOStreams, cmd *cobra.Command) error {
 			ElasticsearchCASHA256: fElasticSearchCASHA256,
 			ElasticsearchInsecure: fElasticSearchInsecure,
 			ServiceToken:          fServiceToken,
+			ServiceTokenFile:      fServiceTokenPath,
```

Comment:
Please change these to have the suffix of `Path` as well, just so it all matches.

Reply:
fleet-server and the elastic-agent-libs both use the suffix `File`;.
If we are going to change it I should also change it in the libs and fleet-server to ensure it's consistent across all usages.
Otherwise I think it's ok to have the `Path` suffix for user interactions (by flags and env vars) in the agent and using `File` for existing serialization

Reply:
We're renaming everything to use `_path`